pkgs/by-name/du/dummyhttp/package.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "dummyhttp";
+  version = "1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "svenstaro";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-MDkyJAVNpCaAomAEweYrQeQWtil8bYo34ko9rAu+VBU=";
+  };
+
+  cargoHash = "sha256-JkA0qW/MQH+XmiD9eiT0s70HxNNYyk9ecBo4k5nUF10=";
+
+  meta = with lib; {
+    description = "Super simple HTTP server that replies a fixed body with a fixed response code";
+    homepage = "https://github.com/svenstaro/dummyhttp";
+    license = with licenses; [ mit ];
+    maintainers = with maintainers; [ ];
```

```suggestion
    maintainers = with maintainers; [ GuillaumeDesforges ];
```

Could you add yourself to maintainers?

---

TBH I'd rather not, but I don't want this to be blocking. What's the policy?

---

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

> `meta.maintainers` must be set for new packages.

Nixpkgs requests to add maintainer name to `meta.maintainers`.